git_wrapper/cherry.py

```diff
@@ -0,0 +1,34 @@
+#! /usr/bin/env python
+'''This module acts as an interface for common git tasks'''
+
+import re
+
+from git_wrapper.base import GitWrapperBase
+
+
+class GitWrapperCherry(GitWrapperBase):
+    '''Provides git cherry functionality'''
+
+    def __init__(self, path='', repo=None):
+        '''Constructor for GitWrapperCherry'''
+        super(GitWrapperCherry, self).__init__(path=path, repo=repo)
+        self.__new_regex = re.compile(r'^\+\s(.*?)\s(.*)')
+        self.__both_regex = re.compile(r'^\-\s(.*?)\s(.*)')
+
```

Comment:
This isn't a review comment, only a question: most of the codebases I worked with were happy with the single underscore convention for private attributes, because the double underscore name mangling was too much trouble I think. I'm wondering about the conventions in this library? Is it used when indicating class-type attributes usually?